content/support/index_administrator.md

```diff
@@ -15,6 +15,7 @@ or email [support@wandb.com](mailto:support@wandb.com).
 
 - [Can I get an academic plan as a student?]({{< relref "academic_plan_student.md" >}})
 - [Can I just log metrics, no code or dataset examples?]({{< relref "just_log_metrics_no_code_dataset_examples.md" >}})
+- [How can I be removed from a team?]({{< relref "how_can_i_be_removed_from_a_team.md" >}})
```

Comment:
If you take the suggestions for updating the FAQ titles, please keep these in sync. :)